roles-in-tech/product-manager.md

```diff
@@ -1,91 +1,112 @@
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [ Who are PM? short video (7.53 min)](https://www.youtube.com/watch?v=2dczveSrsv8) - A short video by Ravijot Chugh which explains 
+who are Product Managers and what do they do?
+- [ Product Manager's Role (10 min)](https://www.altexsoft.com/blog/business/product-management-main-stages-and-product-manager-role/) 
+Use this guide to know more about role of Product Manager.
 
 ### Lesson
 
-Here's text about introducing something and how it works.
-
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
-
-Build on the first information. Have apprentices guess things, do an activity, etc.
-
-Make sure to mention these things:
-
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
+   Go through the slide given in material section to get an idea about Product Management.
+
+ #### Product Management    
+  
+  Product management can be defined as the general business structure within a company that supports and 
+   manages all the activities related to developing, planning, forecasting, marketing and selling a product 
+   all through its lifecycle.
+	
+#### Job of Product Manager
+  
+    Product managers are the official translators within a company, helping commercial teams understand what 
+    the technical, engineering teams mean and vice-versa.
+    
+    Responsibilities that a product manager usually takes are :-
+    - User/market research and analysis
+    - Create a solid Product strategy
+    - Learn to prioritize and re-prioritize various work threads.
+    - Create a product and feature roadmap.
+    - Know how to make a product stable and scalable.
+    - Communicate effectively with Designers, Customers, Engineers and clients.  
+    
+   The Role of PM may change depending on type of product and the company stage, etc.
+
+#### Qualities of PM:-
+    - Strategic thinker
+    - Passion for products
+    - Empathizes with the customer
+    - Aspires to build great user experiences
+    -  Collaborative leader
+    -  Proper Planning and Execution
+ 
 ### Common Mistakes / Misconceptions
-
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
+   
+- Overlook the feedback from customers. 
+- Use too much of tools on a single product which waste both time and efforts.
```

Comment:
Can you rephrase line 81? I’m not sure what this means

Reply:
I changed it. I can also remove it if the point is not clear. I also added a new point.